project/SparkBuild.scala

```diff
@@ -56,9 +56,9 @@ object BuildCommons {
     "tags", "sketch"
   ).map(ProjectRef(buildLocation, _)) ++ sqlProjects ++ streamingProjects
 
-  val optionallyEnabledProjects@Seq(yarn, java8Tests, sparkGangliaLgpl,
+  val optionallyEnabledProjects@Seq(mesos, yarn, java8Tests, sparkGangliaLgpl,
```

Comment:
This is a pattern matching on assignment feature in Scala and the line creates one `optionallyEnabledProjects` value for the entire `Seq` of projects with respective projects being their own values `mesos` etc.